python/hail/expr/expression.py

```diff
@@ -529,6 +530,157 @@ def __ne__(self, other):
             ))
         return self._bin_op("!=", other, tbool)
 
+    def _to_table(self):
+        source = self._indices.source
+        axes = self._indices.axes
+        if not self._aggregations.empty():
+            raise NotImplementedError('cannot convert aggregated expression to table')
+        if source is None:
+            # scalar expression
+            df = hail.Table.range(1) # wat
+            df = df.select(**{'<expr>': self})
```

Comment:
Maybe `Table.parallelize([Struct(**{'<expr>': self})])` is clearer?

Reply:
parallelize takes values, not exprs.

Reply:
Oops, I meant to throw an eval.  Maybe this isn't an improvement.